mention-storer/src/main/java/de/brandwatch/minianalytics/mentionstorer/kafka/Consumer.java

```diff
@@ -0,0 +1,15 @@
+package de.brandwatch.minianalytics.mentionstorer.kafka;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+
+public class Consumer {
+
+    private static final Logger logger = LoggerFactory.getLogger(Consumer.class);
+
+    @KafkaListener(topics = "mentions-topic")
```

Comment:
I'd say just `mentions` instead of `mentions-topic` would be enough?
```suggestion
    @KafkaListener(topics = "mentions")
```